server/paths/manage.ts

```diff
@@ -129,7 +129,7 @@ const v2Manage = {
   outOfServiceBeds: {
     new: outOfServiceBedsPath.path('new'),
     create: outOfServiceBedsPath,
-    premisesIndex: singlePremisesPath.path('out-of-service-beds'),
+    premisesIndex: v2SinglePremisesPath.path('out-of-service-beds').path(':temporality'),
```

Comment:
well, because it's technically its own page (we're using the MOJ sub navigation component, not the govuk tabs component), I thought it made more sense for it to have its own path param. What do you think?

Reply:
No objections, I'm happy to have it like this

Reply:
it's not an issue here but on the all OOS beds page it does lead to slightly peverse URL paths, e.g. `http://localhost:3000/manage/out-of-service-beds/current?temporality=current&sortBy=reason`. Not sure of the obvious way round that though?

Reply:
Yeah, that's less than ideal. Where is the `temporarility` param being added? I am guessing it is unnecessary ?